x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/new_job/wizard_steps.tsx

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment, FC, useState } from 'react';
+
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { EuiStepsHorizontal, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { WIZARD_STEPS } from '../components/step_types';
+import { JOB_TYPE } from '../../common/job_creator/util/constants';
+
+import { TimeRangeStep } from '../components/time_range_step';
+
+import { PickFieldsStep } from '../components/pick_fields_step';
+import { JobDetailsStep } from '../components/job_details_step';
+import { ValidationStep } from '../components/validation_step';
+import { SummaryStep } from '../components/summary_step';
+import { DatafeedStep } from '../components/datafeed_step';
+import { useKibanaContext } from '../../../../contexts/kibana';
+
+interface Props {
+  currentStep: WIZARD_STEPS;
+  highestStep: WIZARD_STEPS;
+  setCurrentStep: React.Dispatch<React.SetStateAction<WIZARD_STEPS>>;
+  disableSteps: boolean;
+  jobType: JOB_TYPE;
+}
+
+export const WizardSteps: FC<Props> = ({
```

Comment:
The numbers at the top look like links, but clicking on them only works if you've moved forwards to that step, and back. Maybe the links should only become links if it's possible to jump to that step?